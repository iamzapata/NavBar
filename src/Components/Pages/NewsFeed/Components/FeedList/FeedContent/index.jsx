import React from "react"
import { string } from "prop-types"

const createContentMarkup = content => ({ __html: content })

const FeedContent = ({ content }) => {
  return (
    <div
      className="FeedContent"
      dangerouslySetInnerHTML={createContentMarkup(content)}
    />
  )
}

FeedContent.propTypes = {
  content: string.isRequired
}

export default FeedContent
