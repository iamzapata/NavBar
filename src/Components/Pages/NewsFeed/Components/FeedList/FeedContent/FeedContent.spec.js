import React from "react"
import { render, cleanup } from "react-testing-library"
import FeedContent from "./index"

describe("<FeedContent />", () => {
  afterEach(cleanup)
  it("Should render self", () => {
    const { container } = render(<FeedContent content="Hey" />)
    expect(container.firstChild).toBeTruthy()
    expect(container.firstChild).toMatchSnapshot()
  })
})
