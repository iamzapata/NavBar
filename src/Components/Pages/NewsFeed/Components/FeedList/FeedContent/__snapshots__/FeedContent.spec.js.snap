// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FeedContent /> Should render self 1`] = `
<div
  class="FeedContent"
>
  Hey
</div>
`;
